src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -777,4 +777,10 @@
   <data name="Windows_Color_Settings.Content" xml:space="preserve">
     <value>Windows color settings</value>
   </data>
+  <data name="PowerRename_BehaviorHeader.Text" xml:space="preserve">
+    <value>Behavior</value>
+  </data>
+  <data name="PowerRename_Toggle_UseBoostLib.Content" xml:space="preserve">
+    <value>Use Boost Library, provides extended features but may behave different</value>
```

Comment:
Let's specify the difference is in the regex part:
```
Use Boost Library (provides extended features but may use different regex syntax)
```

Reply:
I changed the label in `src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw`.

Is the one in [src/modules/powerrename/dll/Resources.resx#144](https://github.com/rhorber/PowerToys/blob/36f8a6d00a8f49bc321046df3fbd0c2d64bb8ec9/src/modules/powerrename/dll/Resources.resx#L144) obsolete? If so, I'll remove it. I added it for [this](https://github.com/rhorber/PowerToys/blob/36f8a6d00a8f49bc321046df3fbd0c2d64bb8ec9/src/modules/powerrename/dll/dllmain.cpp#L237).